lib/mirage.ml

```diff
@@ -1566,10 +1566,22 @@ let get_mode () = Key.(get (get_base_context ()) target)
 
 let libraries_ref = ref []
 let add_to_ocamlfind_libraries l =
+  let pp_v fmt lib = Format.fprintf fmt "\"%s\" ;" lib in
+  let list_pp fmt l = Format.pp_print_list pp_v fmt l in
+  Format.eprintf "add_to_ocamlfind_libraries [%a] is deprecated. \
+      Please instead call `register` with `~libraries:[%a]`.\n"
+      list_pp l list_pp l
+  ;
```

Comment:
Following (I think) @drup's advice at http://stackoverflow.com/questions/37035225/ocaml-attributes I added some annotations to the corresponding `val`s in `mirage_key.mli` and `mirage.mli`, but I don't get any output as a result of having done so.  Unless someone has time to tutor me in remedial deprecation annotations, I'm probably going to release as is.  (It's temporary code, anyway.)